doc/5.reference/binops-tilde-help.pd

```diff
@@ -130,8 +130,8 @@
 #X floatatom 111 228 5 0 0 0 - - - 0;
 #X obj 86 163 sig~ 2;
 #X obj 146 163 sig~ 5;
-#X text 42 15 All of these objects work with multichannel signals. If both inputs have the same amoun of channels \, then operations are done at each corresponding channels. In case the number mismatch \, the input with less channels wrap around and copy the first channels until reaching the number of channels of the other input. This is specially useful to use a single channel value (or a float) to multily to every channel in a multisignal connection., f 67;
 #X text 128 255 value summed to all channels, f 16;
+#X text 42 15 All of these objects work with multichannel signals. If both inputs have the same amoun of channels \, then operations are done at each corresponding channels. In case the number mismatch \, the input with less channels wrap around and copy the first channels until reaching the number of channels of the other input. This is specially useful to use a single channel value (or a float) to apply an operation to every channel in a multisignal connection., f 67;
```

Comment:
`amoun` -> `amount`
`in case the number mismatch` -> `in case the numbers mismatch`
`the input [...] wrap around` -> `the input [...] wraps around`
`specially` -> `especially` (?)
while you are at it: `this is useful to use` sounds a bit weird (but i don't mind very much)

Reply:
done in [2eb805c](https://github.com/pure-data/pure-data/pull/2042/commits/2eb805cd80f4a18fd5adfb778e7adf71ad92c6c7)